include/pulsar/KeySharedPolicy.h

```diff
@@ -98,6 +98,12 @@ class PULSAR_PUBLIC KeySharedPolicy {
      */
     KeySharedPolicy& setStickyRanges(std::initializer_list<StickyRange> ranges);
 
+    /**
+     * @param ranges used with sticky mode
+     */
+    KeySharedPolicy& setStickyRanges(StickyRanges ranges);
```

Comment:
Use `const StickyRanges&` instead of `StickyRanges`

Reply:
Done.